plans/util/integrate_lw_dockerhub.yaml

```diff
@@ -0,0 +1,34 @@
+---
+Dockerhub:
```

Comment:
This file should exist under the util/ path; not as a subdirectory of plans.

Reply:
But this file is a plan. it is the plan to setup the docker hub integration.

Reply:
🤦 My bad @gabeobrien